pkg/compute/init.go

```diff
@@ -355,6 +355,12 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			Comment: description,
 		})
 		if err != nil {
+			if strings.Contains(err.Error(), "Valid values for 'type' are: 'vcl'") {
+				return errors.RemediationError{
+					Inner:       fmt.Errorf("error creating service: you do not have the Compute@Edge feature flag enabled on your Fastly account"),
+					Remediation: "see more at https://fastly.dev/learning/compute/#create-a-new-fastly-account-and-invite-your-collaborators",
```

Comment:
Looking at other remediation messages they all look to start capitalised...

```suggestion
					Remediation: "See more at https://fastly.dev/learning/compute/#create-a-new-fastly-account-and-invite-your-collaborators",
```